context.go

```diff
@@ -108,17 +108,27 @@ func (c *Context) getLoggerModule(name string, tags []string) *module {
 		// First tag wins when setting the logger tag from the config tag
 		// level cache. If there are no tag configs, then fallback to
 		// UNSPECIFIED and inherit the level correctly.
-		if configLevel, ok := c.modulesLabelConfig[tag]; ok && level == UNSPECIFIED {
+		if configLevel, ok := c.modulesTagConfig[tag]; ok && level == UNSPECIFIED {
 			level = configLevel
 		}
 	}
+
+	// As it's not possible to modify the parent's labels, it's safe to copy
```

Comment:
So it's cumulative. Every parent has already inherited from the parent chain above?

Reply:
Correct.